src/fallback.ts

```diff
@@ -18,8 +18,26 @@
 // provide a fast UTF8-only replacement for those browsers that don't support
 // text encoding natively.
 // Also - fun thing! - Node v10 does not have TextDecoder in global scope.
-// eslint-disable-next-line node/no-unsupported-features/node-builtins
-if (typeof TextEncoder === 'undefined' || typeof TextDecoder === 'undefined') {
+import {isBrowser} from './isbrowser';
+let needTextEncoderPolyfill = false;
+
+if (
+  isBrowser() &&
+  // eslint-disable-next-line node/no-unsupported-features/node-builtins
+  (typeof TextEncoder === 'undefined' || typeof TextDecoder === 'undefined')
```

Comment:
Perhaps add a TODO comment here, reminding us to remove this check once we drop Node 10.

Reply:
Added `TODO` for myself.